code/game/objects/items/weapons/melee/energy.dm

```diff
@@ -117,7 +117,7 @@
 
 /obj/item/weapon/melee/energy/glaive
 	name = "energy glaive"
-	desc = "An energized glaive."
+	desc = "An irregular, specialized weapon. Rumor has it the designs for these came from the Hegemony, others insist that this is a Zo'ra design. Whatever the case, in the right hands this is a very nasty weapon with superior reach over typical plasma-based melee weapons."

```

Comment:
Use description_fluff for that. The description should be relatively short and to the point